lib/backend/k8s/k8s_fv_test.go

```diff
@@ -312,4 +318,31 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(len(objs)).To(Equal(0))
 	})
+
+	It("should support setting and getting GlobalConfig", func() {
+		gc := &model.KVPair{
+			Key: model.GlobalConfigKey{
+				Name: "ClusterGUID",
+			},
+			Value: "someguid",
+		}
+		updGC, err := c.Create(gc)
```

Comment:
Nit: Should this be split up with some By() blocks just to indicate what step we're actually doing.